lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -95,6 +97,22 @@ def self.sql(_name, start, finish, _id, payload)
         rescue StandardError => e
           Datadog::Tracer.log.error(e.message)
         end
+
+        def self.instantiation(_name, start, finish, _id, payload)
+          span = get_option(:tracer).trace(
+            'active_record.instantiation',
+            resource: payload.fetch(:class_name),
+            service: get_option(:service_name),
```

Comment:
In the standalone AR, I would prefer to:
* create the span
* check if the service name is our default
* if not, set the service
* if yes, override the span service only if not already set

In this way: `ar_service_name` > parent service > default value (`active_record` let's say).